ExportAs/ExportAs.py

```diff
@@ -77,3 +89,37 @@ def export(self, node, writerFormat):
       logging.info(f"Exported {node.GetName()} to {fileName}")
     else:
       slicer.util.errorDisplay(f"Could not save {node.GetName()} to {fileName}")
+  
+  def exportHardened(self, node, writerFormat):
+    # Clone model node
+    shNode = slicer.vtkMRMLSubjectHierarchyNode.GetSubjectHierarchyNode(slicer.mrmlScene)
+    itemIDToClone = shNode.GetItemByDataNode(node)
+    clonedItemID = slicer.modules.subjecthierarchy.logic().CloneSubjectHierarchyItem(shNode, itemIDToClone)
+    print (clonedItemID)
+    clonedNode = shNode.GetItemDataNode(clonedItemID)
+    
+    # Apply model node transform to cloned node
+    trfNode = slicer.mrmlScene.GetNodeByID(node.GetTransformNodeID())
+    clonedNode.SetAndObserveTransformNodeID(trfNode.GetID())
+
+    # Harden cloned model transform
+    transformLogic = slicer.vtkSlicerTransformLogic()
+    transformLogic.hardenTransform(clonedNode)
+
+    # Save cloned model to file
+    fileName = qt.QFileDialog.getSaveFileName(slicer.util.mainWindow(),
+                                            "Export As...", clonedNode.GetName(), writerFormat)
+    extension = slicer.vtkDataFileFormatHelper.GetFileExtensionFromFormatString(writerFormat)
+    if not fileName.endswith(extension):
+        fileName = fileName + extension
+    if fileName == "":
+      return
+    writerType = slicer.app.coreIOManager().fileWriterFileType(clonedNode)
+    success = slicer.app.coreIOManager().saveNodes(writerType, {"nodeID": clonedNode.GetID(), "fileName": fileName})
+    if success:
+      logging.info(f"Exported {clonedNode.GetName()} to {fileName}")
+    else:
+      slicer.util.errorDisplay(f"Could not save {clonedNode.GetName()} to {fileName}")
+
+    # Remove cloned model Node
+    slicer.mrmlScene.RemoveNode(clonedNode)
```

Comment:
We may need to remove storage node and display nodes of the node as well.

Reply:
Ok